lib/travis/api/v3/renderer.rb

```diff
@@ -49,6 +49,7 @@ def render_value(value, **options)
       when Model                  then render_model(value, **options)
       when ActiveRecord::Relation then render_value(value.to_a, **options)
       when ActiveRecord::Associations::CollectionProxy then render_value(value.to_a, **options)
+      when Travis::Settings::EncryptedValue then value # Should this be value.decrypt ?? If so do we want to add if options[:included].first.public? so we ensure we only decrypt public values?
```

Comment:
done. Though the v3 logic and the v2 logic is to only pass back public values - I think this is a question that will be answered by the upcoming permissions map. I'm going to wait on adding any further permissions logic around this until we have the map.